scribble/scribble.qml

```diff
@@ -61,9 +61,14 @@ Script {
 
         // insert the scribble template image as media file
         var mediaFile = script.insertMediaFile(imagePath, true);
-        var mediaFilePath = script.getApplicationSettingsVariable("legacyLinking") ?
-            mediaFile.replace("file://media", script.currentNoteFolderPath() + "/media") :
-            script.currentNote().fullNoteFileDirPath + "/" + mediaFile;
+        
+        var mediaFilePath;
+        if (script.getApplicationSettingsVariable("legacyLinking") != "false") {
```

Comment:
You can do that also in one code line, like I did. ;)

And can you also please bump the version number and add yourself to the list of authors.

Reply:
I know, all my changes could be written into one line but I'm not a friend of doing so just because you can.
I think it's more readable if you write multiple lines and not only one very long one.

But I can still change it if you want.

Sry totaly forgot the to edit the info.json

Reply:
You can spread the statement over three lines (like I did). You only need one variable declaration, much less redundant. :wink: